api-reference/10 UI Widgets/dxPopover/1 Configuration/shading.md

```diff
@@ -3,3 +3,6 @@ id: dxPopover.Options.shading
 type: Boolean
 default: false
 ---
+Shading applies to the first element with non-static <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/position" target="_blank">position</a> in terms of CSS. The sequence is the following: [container](/Documentation/ApiReference/UI_Widgets/dxPopover/Configuration/#container) => Descendant elements => **window**.
```

Comment:
I went to the MDN document, searched for "non-static", and found nothing. Looks like we should a better job explaining this.

Reply:
There are five types of position: 
1. Static
2. Relative
3. Fixed
4. Absolute
5. Sticky
This is quite a basic thing. I'm sure that web devs have at least minimal understanding of this. 

The list is on the first screen (I don't know why the fixed type is omitted).
![image](https://user-images.githubusercontent.com/33040934/81416955-25295400-9153-11ea-9c27-3121c6d45459.png)

Reply:
The word "static" rings a bell, "non-static" doesn't. We better rephrase it.

BTW, it's not entirely clear what you mean by "the sequence". Is that the order in which the elements are checked for the position value?